### ISSUE TITLE
When serving, asset paths are not reset after modifying with updateDest

### ISSUE THREAD

**Issue by valpackett** [lumeland/lume#289]
When running a dev server, something like this:

```js
let cnt = 1

site
  .loadAssets([".css"])
  .process([".css"], (page) => {
    cnt += 1
    page.updateDest({ path: page.dest.path + `.${cnt}` })
  })
```

keeps modifying *the same* path string, building up new additions all the time, i.e. like `s.2.3.4.5.css`:

![image](https://user-images.githubusercontent.com/208340/194784399-94f75dbe-bb3a-4d55-b6bd-68edab9e3b5a.png)

This is not the expected behavior :)

(version 1.12.0)

**Comment by oscarotero** [comment]
Yes, this is how it works now, but I guess it's not what users expect.
I have to think in a way to reset the page data to prevent this incremental changes, without affecting to the performance.

But this may take a while. For now you have to replace the old value with the new.

**Comment by oscarotero** [comment]
Fixed. It will be available in Lume 1.13.0
